libtransact/src/database/mod.rs

```diff
@@ -33,7 +33,7 @@ pub mod btree;
 pub mod error;
 pub mod lmdb;
 
-use crate::database::error::DatabaseError;
```

Comment:
typo in commit message: integegration -> integration

Reply:
Fixed